lighty-examples/lighty-gnmi-community-restconf-app/README.md

```diff
@@ -0,0 +1,81 @@
+configured by [this](src/main/assembly/resources/simulator/initialJsonData.json) json file.
+For communication with gNMI device is required to use TLS communication with certificates and authorized
+by username and password.
+
+This application starts:
+* [Lighty.io Controller](https://github.com/PANTHEONtech/lighty/tree/master/lighty-core/lighty-controller)
+* [lighty.io RESTCONF](https://github.com/PANTHEONtech/lighty/tree/master/lighty-modules/lighty-restconf-nb-community)
+* [lighty.io gNMI](https://github.com/PANTHEONtech/lighty/tree/master/lighty-modules/lighty-gnmi/lighty-gnmi-sb)
+* [lighty.io gNMI device simulator](https://github.com/PANTHEONtech/lighty/tree/14.0.x/lighty-modules/lighty-gnmi/lighty-gnmi-device-simulator)
+
+## Prerequisites
+In order to build and start and use the lighty.io gNMI/RESTCONF application locally, you need:
+* Java 11 or later
+* Maven 3.5.4 or later
+* Postman v7.36.5. or later
+
+## Build and start
+To build and start the lighty.io gNMI/RESTCONF application in your local environment, follow these steps:
+1. Build the application using maven
+   `mvn clean install`
+
+2. Unpack the application ZIP distribution created in the _lighty-gnmi-community-restconf-app/target_ called
+   `lighty-gnmi-community-restconf-app-<version>-bin.zip`
+
+3. Start the application by:
+   - Running it's _.jar_ file `java -jar lighty-gnmi-community-restconf-app-<version>-SNAPSHOT.jar`
+   - Or with provided script `./start-controller.sh`
+
+## Example of using lighty.io gNMI/RESTCONF application
+This example show how to connect gNMI device and perform basic CRUD operation on the device. All RESTCONF request
+used in this example are provided in [postman-collection](lighty.io gNMI-RESTCONF application.postman_collection.json).
+
+ - ### Add client certificates to lighty.io gNMI keystore
+Used certificates can be found [here](src/main/assembly/resources/certificates). To keystore is added only client
+certificates. Adding required certificates for gNMI device to lighty.io gNMI application is performed by
+postman request `'Add Keystore'`.
+
+ - ### Connect gNMI device
+Device connection is performed by request `'Connet device'`. In the body of this request is contained identifier
+for keystore, device information, extension parameters and basic authorization required by device. When device
+is successfully connected to application, then these logs should be visible:
+```
+ INFO [gnmi_executor-1] (GnmiMountPointRegistrator.java:52) - Mount point for node gnmi-simulator created: {closed=false, instance=org.opendaylight.mdsal.dom.spi.SimpleDOMMountPoint@60112bc5}
+ INFO [gnmi_executor-0] (GnmiNodeListener.java:105) - Connection with node Uri{_value=gnmi-simulator} established successfully
+```
+
+Device state can be also checked by request `'Get gnmi-simulator node'`. Inside this request can be found information about
```

Comment:
information about created device can be found inside this request

Reply:
done